### PR TITLE
Fake Exchange Web Access and fake ABSA page

### PR DESCRIPTION
**Pull request by wagonza**

### CODE REVIEW

**Comment by spirillen** [comment]
Before I would merge this one... but it appears to be rather dead that domain...

```shell
drill onlinesupportsystem.support -T
.       518400  IN      NS      a.root-servers.net.
support.        172800  IN      NS      demand.gamma.aridns.net.au.
support.        86400   IN      SOA     demand.alpha.aridns.net.au.
```

The WHOIS is active, but that's about it.
```

pyfunceble -a -d onlinesupportsystem.support --dns 127.0.0.1:5302

Subject                                                                                              Status      Source     Expiration Date   HTTP Code  Checker      
---------------------------------------------------------------------------------------------------- ----------- ---------- ----------------- ---------- -------------
onlinesupportsystem.support                                                                          ACTIVE      WHOIS      23-jun-2022       Unknown    AVAILABILITY 
```

Do you have any information's, that can help investigate this domain??

File: add-link [hunk 1]
```diff
@@ -32,3 +32,5 @@ http://moderninterior.co.in/unipage/Pennsylvania's/Unemployment/
 https://service-survey.vip/warranty
 https://87j2a9ue06.prizechariot.com/?aff_id=1087&offer_id=210&transaction_id=102b6446b450e7a074e39dd2c9c0be&aff_sub1=60d9f1738d4d610001624e74&aff_sub2=168&aff_sub3=&link=1087&ship-first-name=%7Bship-first-name%7D&ship-last-name=%7Bship-last-name%7D&ship-email=%7Bship-email%7D&ship-phone=%7Bship-phone%7D&ship-address=%7Bship-address%7D&ship-city=%7Bship-city%7D&ship-zip-code=%7Bship-zip-code%7D&ship-country=US&ship-state=%7Bship-state%7D&mc=%3F
 https://restore950339-coinbase-us.web.app/signin
+https://onlinesupportsystem.support/
+https://coupons4you.net/cmc.bern/ssbcn.php
```
Comment (spirillen) [inline add-link:36]: I can't reproduce this one...

Anyone else ho can re-produce this one?

image

Comment (wagonza) [reply add-link:36]: Yeah it since has been removed

**Review by wagonza — COMMENTED** [review]
This domain is no longer active

File: add-link (hunk 1 above)
Comment (spirillen) [reply add-link]: that means something is working the way we like it :smiley:

**Review by spirillen — APPROVED** [review]

**Pull request merged**